backend/src/test/java/turip/favorite/api/FavoriteApiTest.java

```diff
@@ -0,0 +1,163 @@
+            jdbcTemplate.update(
+                    "INSERT INTO creator (profile_image, channel_name) VALUES ('https://image.example.com/creator1.jpg', 'TravelMate')");
+            jdbcTemplate.update("INSERT INTO country (name) VALUES ('korea')");
+            jdbcTemplate.update("INSERT INTO city (name, country_id, province_id) VALUES ('seoul', 1, null)");
+            jdbcTemplate.update("INSERT INTO category (name) VALUES ('관광지')");
+            jdbcTemplate.update(
+                    "INSERT INTO place (name, url, address, latitude, longitude) VALUES ('경복궁','https://naver.me/test','경복궁 도로명 주소',37.1234,126.1234)");
+            jdbcTemplate.update("INSERT INTO place_category (place_id, category_id) VALUES (1, 1)");
+            jdbcTemplate.update(
+                    "INSERT INTO content (creator_id, city_id, url, title, uploaded_date) VALUES (1, 1, 'https://youtube.com/watch?v=test123', '서울 여행', '2025-06-01')");
+            jdbcTemplate.update(
+                    "INSERT INTO trip_course (content_id, place_id, visit_day, visit_order) VALUES (1, 1, 1, 1)");
+            jdbcTemplate.update("INSERT INTO member (device_fid) VALUES ('testDeviceFid')");
+            jdbcTemplate.update(
+                    "INSERT INTO favorite (member_id, content_id, created_at) VALUES (1, 1, CURRENT_TIMESTAMP)");
+
+            // when & then
+            Map<String, String> request = new HashMap<>(Map.of("contentId", "1"));
+            RestAssured.given().port(port)
+                    .header("device-fid", "testDeviceFid")
+                    .contentType(ContentType.JSON)
+                    .body(request)
+                    .when().post("/favorites")
+                    .then()
+                    .statusCode(400);
+        }
+    }
+
+    @DisplayName("/favorites DELETE 찜 삭제 테스트")
+    @Nested
+    class deleteFavorite {
+
+        @DisplayName("성공 시 204 No Content를 응답한다")
+        @Test
+        void removeFavorite() {
+            // given
+            jdbcTemplate.update(
+                    "INSERT INTO creator (profile_image, channel_name) VALUES ('https://image.example.com/creator.jpg', 'Travel')");
+            jdbcTemplate.update("INSERT INTO country (name) VALUES ('korea')");
+            jdbcTemplate.update("INSERT INTO city (name, country_id, province_id) VALUES ('seoul', 1, null)");
+            jdbcTemplate.update(
+                    "INSERT INTO content (creator_id, city_id, url, title, uploaded_date) VALUES (1, 1, 'https://youtube.com/watch?v=deleteTest', '삭제 테스트 영상', '2025-06-01')");
+            jdbcTemplate.update("INSERT INTO member (device_fid) VALUES ('testDeviceFid')");
+            jdbcTemplate.update(
+                    "INSERT INTO favorite (member_id, content_id, created_at) VALUES (1, 1, CURRENT_TIMESTAMP)");
+
+            // when & then
+            RestAssured.given().port(port)
+                    .header("device-fid", "testDeviceFid")
+                    .when().delete("/favorites?contentId=1")
```

Comment:
a: 저는 쿼리 파라미터는 명시적으로 나타내는게 테스트 가독성에 좋다고 생각하는데, 어떻게 생각하시나용??

```suggestion
.queryParam("contentId", 1)
.when().delete("/favorites")
```